src/VisualStudio/Core/Def/Implementation/TableDataSource/AbstractTableEntriesSnapshot.cs

```diff
@@ -211,5 +216,75 @@ public void StartCaching()
         public void StopCaching()
         {
         }
+
+        protected static bool CanCreateDetailsContent(int index, Func<int, DiagnosticTableItem?> getDiagnosticTableItem)
```

Comment:
Entire new block of code in this file is just moved down without any changes from a sub-type.